src/main/java/unrc/dose/BellyService.java

```diff
@@ -0,0 +1,63 @@
+package unrc.dose;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/** This is a Service to store a bunch of Bellies together. */
+public final class BellyService {
+
+    /** Bellies store. */
+    private static Map<String, Belly> bellies = new HashMap<String, Belly>();
+
+    /** Atomic counter. */
+    private static final AtomicInteger COUNT = new AtomicInteger(0);
+
+    /** Returns a particular belly.
+     * @param id String
+     * @return Belly
+    */
+    public Belly findById(final String id) {
+        return (Belly) bellies.get(id);
```

Comment:
There should be an argument check (value not null nor empty) and `bellies.get(id)` can return `null`, this may be the expected behavior but maybe using a wrapper like `Optional` or a Haskell's Maybe equivalent might be a better option. One could also use a `BellyNotFoundException` although I don't think is the best option for this particular method.